src/configFirebase.env

```diff
@@ -0,0 +1,8 @@
+export const firebaseConfig = {
+  apiKey: 'AIzaSyCYHV8emihSZclfdX0XFdWRbhKwhSrnBP0',
+  authDomain: 'dims-17.firebaseapp.com',
+  projectId: 'dims-17',
+  storageBucket: 'dims-17.appspot.com',
+  messagingSenderId: '649779985987',
+  appId: '1:649779985987:web:03aaa1dab65e7a55c61ef6',
```

Comment:
1) `.env` file - check in gitignore
2) REACT_APP_API_KEY='AIzaSyCYHV8emihSZclfdX0XFdWRbhKwhSrnBP0'
3) delete configFirebase.env from pull-request
4) process.env.REACT_APP_API_KEY - call the variable in you config

Reply:
Ok, thanks, I understood.